lib/equation/Equation.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const math = require('mathjs');
+math.config({number: 'Fraction'});
```

Comment:
I wonder if there's a way to just set this globally (cause you're just setting config whenever you `require` `mathjs` right?